src/commands/call.ts

```diff
@@ -93,7 +115,65 @@ export class ActorCallCommand extends ApifyCommand<typeof ActorCallCommand> {
 			type: 'Actor',
 			waitForFinishMillis,
 			inputOverride: inputOverride?.input,
+			silent: this.flags.silent,
 		});
+
+		if (this.flags.outputDataset) {
+			// TODO: cleaner way to do this
+			while (!TerminalStatuses.includes(run.status as never)) {
```

Comment:
This part effectively waits for the Actor to finish, right? What's our opinion on CLI progress bars / throbbers with status message, number of dataset results etc.? IMO as an user, I'd love to see that something is happening instead of a blank shell line (I've had my fair share of CLI tools just freeze on me, I usually kill them like 10 seconds in).

(Sorry if this is not relevant, I didn't have the time to run this, just reading the code).

Reply:
It technically won't be an issue unless you run with `--silent` (cuz otherwise you'll see the stream of logs from the actor run). But yeah... definitely something we may need to circle back and think about 👍

Reply:
Oh right, that's exactly why I should have ran it first 🤦🏽‍♂️ Sorry for the fuss, that makes sense. 

btw definitely a good point about the silent mode 👍🏽 although maybe that should be ...completely silent? I can imagine some ppl not being very happy about the terminal escape sequences moving the cursor back and forth, especially if you opt-in a "silent" mode.

Reply:
Currently it is fully silent (no stderr logs at all, and stdout will only have the results)!

Reply:
I perceive the total silence as intimidating, too. :) The idea is that it should only return platform events to `stderr` in the silent mode, so it will suppress logging by Actor author code and won't break integrations to `stdout`. 

How does that sound to you? We can add it later. This functionality demonstrates the benefits of using Actors end-to-end from a CLI in a future blog post.

Reply:
I'm a bit confused what you mean by that @netmilk... Should the stderr messages still be emitted when `--silent` is passed or no?